### PR TITLE
Schema migration for changing 'metadata' to 'exec_state'

### PR DESCRIPTION
**Pull request by likawind**
## Describe your changes and why you are making these changes
This PR adds schema migration for `operator_result` table by changing `metadata` to `exec_state` based on changes introduced in #159 . We added 3 new versions to add new col, backfill, and drop old col. After this change the schema will become 12.

## Related issue number (if any)
ENG-1249

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] #161  I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] #168  If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Review by cw75 — COMMENTED** [review]
@likawind I was under the impression that SQLite doesn't support dropping columns, but according to [here](https://stackoverflow.com/a/5987838) the recent version does seem to support this. Have you verified that this works? Also reminder to update the schema change tracker [doc](https://www.notion.so/aqueducthq/Database-Schema-Change-Tracker-c861ce6ba45b43ad87fe7d91eee49df3) when you get a chance. Thx!

File: src/golang/cmd/migrator/versions/000011_exec_state_column_backfill/queries.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+type opResultWithMetadata struct {
+	Id       uuid.UUID              `db:"id" json:"id"`
+	Status   shared.ExecutionStatus `db:"status" json:"status"`
+	Metadata NullMetadata           `db:"metadata" json:"metadata"`
+}
+
+type opResultWithExecState struct {
+	Id        uuid.UUID                 `db:"id" json:"id"`
+	Status    shared.ExecutionStatus    `db:"status" json:"status"`
+	ExecState shared.NullExecutionState `db:"exec_state" json:"exec_state"`
```
Comment (saurav-c) [inline src/golang/cmd/migrator/versions/000011_exec_state_column_backfill/queries.go:21]: Won't this no longer work if we modify `shared.NullExecutionState` in the future? This is why we have to redefine the structs to the state they were in at the time of this migration.

Comment (likawind) [reply src/golang/cmd/migrator/versions/000011_exec_state_column_backfill/queries.go]: Good point, fixed

**Comment by likawind** [comment]
Integration test was successful. Current failure due to v0.0.4 release, will rerun after rebase

**Review by saurav-c — APPROVED** [review]
LGTM

**Pull request merged**